### ISSUE TITLE
Documenting All Classes

### ISSUE THREAD

**Issue by MarkVaughn** [stripe/stripe-php#250]
I'd volunteer to add class level phpdoc blocks to all classes.

Example: 
Customer.php
```php
<?php

namespace Stripe;

/**
 * Class Customer
 *
 * @property string $id
 * @property string $object
 * @property int $account_balance
 * @property string $business_vat_id
 * @property string $created
 * @property string $currency
 * @property string $default_source
 * @property bool $delinquent
 * @property string $description
 * @property mixed $discount
 * @property string $email
 * @property bool $livemode
 * @property array $metadata
 * @property mixed $shipping
 * @property array $sources
 * @property array $subscriptions
 *
 * @package Stripe
 */
class Customer extends ApiResource
```

This would help with development in IDE's such as phpstorm
![autocompletion](https://cloud.githubusercontent.com/assets/39970/15052384/691afa02-12cb-11e6-9ada-9a19d7cc3e80.png)

Again I'd volunteer to do it and put up a PR, but I don't want to do it all and then get some reason why you guys would reject it for some reason.
Let me know

**Comment by arcanedev-maroc** [comment]
Check this one: [ARCANEDEV/Stripe](https://github.com/ARCANEDEV/Stripe)

**Comment by brandur** [comment]
Hi @MarkVaughn, and I guess the proposal here is mostly to just get types in rather than an additional description as well?

My only fear here is that we don't really have a great way of ensuring the accuracy or timeliness of update of such an addition. That said, I wouldn't be too adverse to the idea if you want to go for a pull request.

**Comment by MarkVaughn** [comment]
Yea, just some sort of bare minimum better than nothing.
Also knowing what fields are potentially available. I'll get on it then. 
Cheers

**Comment by brandur** [comment]
>  Also knowing what fields are potentially available. I'll get on it then. 

Makes sense! And cool, thanks for the effort here.

**Comment by jarrodbell** [comment]
Any update on this @MarkVaughn ? I'd love this for working with the API in PhpStorm

**Comment by MarkVaughn** [comment]
@jarrodbell @brandur made a pr, but apparently it is "useless"

**Comment by GrahamCampbell** [comment]
> made a pr, but apparently it is "useless"

That wasn't what I said. I made an inline comment on a specific line.

**Comment by MarkVaughn** [comment]
Thats fine, the two comments just had very little information and were pretty kurt and I've seen the style of my comments throughout many Open source projects, prefacing with the "Class/Interface/Trait name" before a block of descriptive text, then the methods and properties followed by author/package/link etc...
In my mind this is better than the no documentation style currently in place so unless you have improvements I don't see the harm in having consistency
e.g
https://github.com/elastic/elasticsearch-php/blob/master/src/Elasticsearch/Endpoints/Exists.php
https://github.com/facebook/facebook-php-sdk-v4/blob/master/src/Facebook/FacebookClient.php

**Comment by brandur** [comment]
Fix was in #268. Closing this out!
